src/Behat/Behat/Context/Snippet/Generator/ContextSnippetGenerator.php

```diff
@@ -161,17 +161,18 @@ private function getUsedClasses(StepNode $step): array
     {
         $usedClasses = [PendingException::class];
 
-        $usedClasses[] = match ($step->getKeywordType()) {
+        $definitionClass = match ($step->getKeywordType()) {
```

Comment:
The match statement did not contemplate the case where the $definitionClass could be something else

Reply:
this should be solved in behat/gherkin by defining a more precise type for this getter (there is only 3 valid keyword types)

Reply:
OK, until this is solved at that level, I added an assertion here